plugins/webp-uploads/settings.php

```diff
@@ -35,10 +50,24 @@ function webp_uploads_register_media_settings_field() {
  * @since 1.0.0
  */
 function webp_uploads_add_media_settings_field() {
+
+	// If AVIF is supported, add a dropdown to select the output format between AVIF and WebP.
+	if ( wp_image_editor_supports( array( 'mime_type' => 'image/avif' ) ) ) {
```

Comment:
Should some field still be printed even when AVIF is not supported by the server to inform the user that this is why AVIF isn't available? I guess there is a Site Health test for that, but only if Performance Lab is installed, right?

Reply:
> Should some field still be printed even when AVIF is not supported by the server

I guess we could show a dropdown with WebP and AVIF, but AVIF is disabled, wdyt? (Thats actually what I do in my https://github.com/adamsilverstein/modern-images-wp plugin)

> I guess there is a Site Health test for that, but only if Performance Lab is installed, right?

Right, I see that landed in https://github.com/WordPress/performance/pull/1177 🎉

Reply:
Having AVIF alone disabled may not be enough. It might need a separate inline WP notice, since when there is just one disabled option and one enabled/selected option, it's not entirely clear that the disabled option is disabled from the gray text alone:

[Screen recording 2024-05-01 14.31.09.webm](https://github.com/WordPress/performance/assets/134745/dc5b5c4b-a351-46e7-bbdd-4347534d41b6)

Reply:
I can't see the screenshot for some reason, but I think that makes sense. Maybe we can do both the disabled option and the inline notice.

Reply:
I added both an inline notice and disabled "AVIF" in the dropdown in [`88dedcb` (#1176)](https://github.com/WordPress/performance/pull/1176/commits/88dedcb2b40ef9d7a128fa5f7abf5ba9a16f79fe) and [`3896293` (#1176)](https://github.com/WordPress/performance/pull/1176/commits/3896293ca512c7680939555801b9ed8b95f58033)